Tests/Common/ASTestCase.mm

```diff
@@ -64,12 +64,12 @@ - (void)tearDown
 
   for (OCMockObject *mockObject in registeredMockObjects) {
     OCMVerifyAll(mockObject);
-    [mockObject stopMocking];
+   // [mockObject stopMocking];
```

Comment:
Hi @dmaclach Where is it being done elsewhere? It's not clear to me from your comment. Thanks (and hello)!

Reply:
Hey Greg!

My fault.. fat fingered an auto complete. I'm going to drop this PR and replace it with #1822 1822